eoscostarica/eos-rate
# upgrade front page UI

### GH Issue

# Upgrade home page.

### Steps to test

1. run the project
2. see the home page

### Checklist
